src/main/java/onboarding/Problem1.java

```diff
@@ -1,10 +1,55 @@
 package onboarding;
 
 import java.util.List;
+import java.util.function.BiFunction;
 
 class Problem1 {
+    private static final int DRAW = 0;
+    private static final int POBI_WIN = 1;
+    private static final int CRONG_WIN = 2;
+    private static final int EXCEPTION = -1;
+
     public static int solution(List<Integer> pobi, List<Integer> crong) {
-        int answer = Integer.MAX_VALUE;
-        return answer;
+        if (validationPages(pobi) || validationPages(crong)) {
+            return EXCEPTION;
+        }
+        int pobiMaxValue = getMaxValue(pobi);
+        int crongMaxValue = getMaxValue(crong);
+
```

Comment:
게임의 결과를 반환해주는 메소드를 따로 선언하면 어떨까요?
```suggestion
        return getScore(pobiMaxValue, crongMaxValue);
```